build-system/global-configs/canary-config.json

```diff
@@ -34,7 +34,7 @@
   "swg-gpay-api": 1,
   "hidden-mutation-observer": 1,
   "ios-fixed-no-transfer": 1,
-  "layoutbox-invalidate-on-scroll": 1,
+  "layoutbox-invalidate-on-scroll": 0.05,
```

Comment:
Why ramp down canary?

Reply:
So that canary and rc will have the same fractional experiment. May not be required, since rc and control will have the same fraction.